### PR TITLE
PROJECT-003: Create User Authentication Functionality

### PR DESCRIPTION
**Pull request by andrewwashko**
Completed FE and BE User Authentication functionality. Edited React components to better reflect initial wireframing.

### CODE REVIEW

File: book_recommender_app/models.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+class App_User(AbstractUser):
+    email = models.EmailField(blank = False, null = False, unique = True)
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = []
```
Comment (nfrankeln) [inline book_recommender_app/models.py:7]: is username/email not required?

File: book_recommender_react_project/src/components/SignUp.jsx
Comment (nfrankeln) [inline book_recommender_react_project/src/components/SignUp.jsx:1]: look up react-hook-forms for future reference!

**Review by nfrankeln — APPROVED** [review]

File: book_recommender_app/models.py (hunk 1 above)
Comment (andrewwashko) [reply book_recommender_app/models.py:7]: The [Django docs](https://docs.djangoproject.com/en/4.2/topics/auth/customizing/) mentioned that the username field and password are required by default, so I left it blank!

File: book_recommender_react_project/src/components/SignUp.jsx
Comment (andrewwashko) [reply book_recommender_react_project/src/components/SignUp.jsx:1]: Awesome! Thank you for the resource.

**Pull request merged**